control-plane/pkg/reconciler/broker/broker_lifecycle.go

```diff
@@ -34,7 +34,7 @@ const (
 	ConditionConfigMapUpdated apis.ConditionType = "ConfigMapUpdated"
 )
 
-var conditionSet = apis.NewLivingConditionSet(
+var ConditionSet = apis.NewLivingConditionSet(
 	ConditionReady,
```

Comment:
NewLivingConditionSet - will include a top level 'Ready' - what you're supplying here are dependent conditions

Reply:
Ok, thank you.